.github/workflows/ontology_diff.yml

```diff
@@ -0,0 +1,70 @@
+name: Post Markdown Comment on Pull Request
+
+on:
+    workflow_dispatch:
+    pull_request:
+        types: [opened, synchronize]
+
+jobs:
+    post-comment:
+        runs-on: ubuntu-latest
+
+        steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - name: Set up Python 3.10
+          uses: actions/setup-python@v5
+          with:
+            python-version: '3.10'
+
+        - name: Install oaklib
+          run: pip install git+https://github.com/INCATools/ontology-access-kit.git@release-notes-emit
```

Comment:
This should change before merging (I know this is for testing now)

Reply:
I had already made a note above in the description about this.